x/shared/helpers/service_test.go

```diff
@@ -28,6 +32,78 @@ func TestIsValidServiceId(t *testing.T) {
 	}
 }
 
+func TestIsValidServiceName(t *testing.T) {
+	tests := []struct {
+		input    string
+		expected bool
+	}{
```

Comment:
I think this would read nicer if we cleaned it up like the rest of the table-driven tests.

Reply:
Deferring to #150, and will rebase on top of that